src/bp/core/services/nlu/nlu-engine.ts

```diff
@@ -0,0 +1,26 @@
+import { IO } from 'botpress/sdk'
+import { injectable } from 'inversify'
+
+@injectable()
+export class NLUEngine {
+  constructor() {}
+
+  public processEvent(event: IO.Event): Promise<IO.Event> {
```

Comment:
I believe this would be the place I make the prediction if the nlu module was moved into the core.